fuckingsite_c3/src/TaskList/comp.js

```diff
@@ -1,10 +1,20 @@
 import React, { Component } from 'react';
 import './comp.style.scss';
+import Task from '../Task/comp.js';
 
 class TaskList extends Component {
     render(){
+        const tasks = this.props.tasks;
+        const removeTask = this.props.removeTask;
         return (
-            <ul className = "task--list">{this.props.children}</ul>
+            <ul className = "task--list">
+              {tasks.map((task,id) => {
+                  task.id = id;
+                  return ( 
+                    <Task onClickHandler={(id) => removeTask(id)} task={task} />
```

Comment:
rather than create several props like id, key, taskname, it is easier to simply add the `id` to `task` as seen in line 12, then pass the task object to be parsed.